docs/security-compliance/personal-data-location.md

```diff
@@ -0,0 +1,35 @@
+---
+id: personal-data-location
+title: Personal data location and homing
+sidebar_label: Personal data storage
+---
+
+Ory Network is globally distributed for maximum resilience and minimum latency. Check out our
+[blog post](https://www.ory.sh/global-identity-and-access-management-multi-region/) for details on how this works.
+
+As discussed in that blog post, replicating personal data (also known as personally identifiable data or PII) across the globe is
+prohibited or made difficult by legislation like GDPR and would your user's control over their data.
```

Comment:
I think there is a word missing here? At least to me, the would doesn't make sense?

Reply:
ah you were faster ;-)